prepareLocalDevelopment.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
maybe we should name this prepareAndroidLocalDevelopment since it doesn't do ios?

Reply:
very minor, but I think bash scripts are more often than not named using snake case

Reply:
also, let's put it in a folder to keep the root folder clean, especially since this is only for development purposes and not for the plugin itself